sections/main-footer.liquid

```diff
@@ -2,17 +2,28 @@
 
 {% style %}
   :root  {
-    --yc-footer-background-color: {%- if section.settings.footer_background_color -%}
-      {{ section.settings.footer_background_color.hex }}
-    {%- else -%}
-      var(--yc-primary-color){%- endif -%};
+    --yc-footer-background-color: {{ section.settings.footer_background_color.hex }};
     --yc-footer-text-color: {{ section.settings.footer_text_color.hex }};
-    --yc-footer-copyright-background: {{ section.settings.footer_copyright_background.hex }};
+    --yc-footer-border-color: {{ section.settings.footer_border_color.hex }};
   }
 {% endstyle %}
 
 <footer class='yc-footer'>
   <div class="columns container">
+    {%- if section.settings.footer_logo.src -%}
+      <div class='footer-logo column'>
+        <img loading='lazy' src='{{ section.settings.footer_logo.src }}'>
+      </div>
+    {%- endif -%}
+    {%- if section.settings.mention_and_social_media -%}
+      <div class='social-media-links column'>
+        {%- for block in section.blocks -%}
+          <a class='link' href="{{ block.settings.social_media_url }}" target="_blank">
+            <img loading='lazy' class='image' src="{{ block.settings.social_media_icon_image.src }}">
+          </a>
+        {%- endfor %}
+      </div>
+    {%- endif -%}
```

Comment:
If the user doesn't add a social media block, there is no need to display an empty link. 

<img width="832" alt="Screen Shot 2023-03-21 at 10 13 17" src="https://user-images.githubusercontent.com/108662497/226576830-82f793c8-8711-41c8-9281-03cf9cf752a4.png">

Reply:
yeah sure until we fix the theme editor doesn't have the ability to delete a social media block in the footer

Reply:
I have set a default image on icons, if the user didn't add a social media icon until we fix the issue in the theme editor